### PR TITLE
Switch to an http_archive for io_k8s_repo_infra

### PR DESCRIPTION
**Pull request by fejta**

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+http_archive(
     name = "io_k8s_repo_infra",
-    commit = "e7bd7a7c8b3f900d5344bf5909c60f890f9719aa",
-    remote = "https://github.com/kubernetes/repo-infra.git",
-    shallow_since = "1571503417 -0700",
-    # tag = "v0.0.1-alpha.1",
+    sha256 = "5ee2a8e306af0aaf2844b5e2c79b5f3f53fc9ce3532233f0615b8d0265902b2a",
+    strip_prefix = "repo-infra-0.0.1-alpha.1",
+    urls = [
+        # "https://mirror.bazel.build/github.com/kubernetes/repo-infra/archive/v0.0.1-alpha.1.tar.gz",
```
Comment (michelle192837) [inline WORKSPACE]: Remove?

Comment (fejta) [reply WORKSPACE]: Done

**Comment by fejta** [comment]
Can I get another review?

**Pull request merged**